common/protob/messages-common.proto

```diff
@@ -110,31 +111,16 @@ message PinMatrixAck {
  * @next PassphraseAck
  */
 message PassphraseRequest {
-    optional bool on_device = 1;    // passphrase is being entered on the device
+//    optional bool on_device = 1;  // DEPRECATED
 }
 
 /**
  * Request: Send passphrase back
- * @next PassphraseStateRequest
+ * @auxend
  */
 message PassphraseAck {
     optional string passphrase = 1;
-    optional bytes state = 2;       // expected device state
-}
-
-/**
- * Response: Device awaits passphrase state
- * @next PassphraseStateAck
- */
-message PassphraseStateRequest {
-    optional bytes state = 1;       // actual device state
-}
-
-/**
- * Request: Send passphrase state back
- * @auxend
- */
-message PassphraseStateAck {
+    optional bool on_device = 2;    // user wants to enter passphrase on the device
```

Comment:
the diff makes it difficult to see, but we changed a field type here.
orig: `optional bytes state = 2`
new: `optional bool on_device = 2`
it really should be 3

Reply:
Fixed in 56dc952495eac8ead7e750e60eded7096e41f6b2